test-dev/dev.test.ts

```diff
@@ -35,8 +44,10 @@ describe('snowpack dev', () => {
       {cwd},
     );
 
-    snowpackProcess.stdout.pipe(process.stdout);
-    snowpackProcess.stderr.pipe(process.stderr);
+    const streamLogFile = fs.createWriteStream(debugFilePath);
```

Comment:
what's the reason behind these changes?

Reply:
The reasoning is basically to make the logging output more understanable/readable. As you might now, if you start i.e. `snowpack dev` the console is completely cleared and thus you can't see potentional errors emitted by jest tests. Hence I introduced a clear seperation of the debug logs of snowpack and the usual console output by jest.

I hope this is an understandable reasoning to make this decision as it was pretty hard to debug this test without having this separation.

Reply:
makes sense! thanks for clarifying

Reply:
We have `--output stream` for this exact reason, i'll look into enabling that here after merging